drivers/espi/espi_mchp_xec.c

```diff
@@ -204,9 +201,10 @@ static const struct xec_signal vw_tbl[] = {
 					     ESPI_MASTER_TO_SLAVE},
 };
 
-static uint32_t slave_rx_mem[MAX_OOB_BUFFER_SIZE];
-static uint32_t slave_tx_mem[MAX_OOB_BUFFER_SIZE];
-static uint32_t slave_mem[MAX_SLAVE_BUFFER_SIZE];
+/* Buffer size are expressed in bytes */
+static uint32_t target_rx_mem[CONFIG_ESPI_OOB_BUFFER_SIZE >> 2];
+static uint32_t target_tx_mem[CONFIG_ESPI_OOB_BUFFER_SIZE >> 2];
```

Comment:
this shouldnt be coved with #ifdef CONFIG_ESPI_OOB_CHANNEL ?
assume that espi driver comliped with oob disabled

Reply:
Yes, it could help reduce further the RAM when OOB channel is not enabled
Not the case in most system but i see your point.